src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -353,16 +383,11 @@ public void Read(Span<PinValuePair> pinValuePairs)
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="GpioController"/> class that will use the specified numbering scheme.
-        /// The controller will default to use the driver that best applies given the platform the program is executing on.
+        /// Tries to create the GPIO driver that best matches the current hardware
         /// </summary>
-        /// <param name="numberingScheme">The numbering scheme used to represent pins provided by the controller.</param>
-        public GpioController(PinNumberingScheme numberingScheme)
-            : this(numberingScheme, GetBestDriverForBoard())
-        {
-        }
-
-        private static GpioDriver GetBestDriverForBoard()
+        /// <returns>An instance of a GpioDriver that best matches the current hardware</returns>
+        /// <exception cref="PlatformNotSupportedException">No matching driver could be found</exception>
+        protected static GpioDriver GetBestDriverForBoard()
```

Comment:
I think perhaps this should be just GpioDriver.Create

Reply:
cc: @joperezr

Reply:
How would we know which driver to create?